client/styles/global.js

```diff
@@ -51,17 +51,18 @@ export const FlexContainer = styled.div`
   flex-direction: ${({ flexDirection }) => flexDirection || null};
   justify-content: space-around;
   align-items: center;
-  margin-bottom: 20px;
+  margin: ${({ margin }) => margin || "20px 20px"};
```

Comment:
here you set margin 20px anyway ? you can simply pass 20px one time :)
```suggestion
  margin: ${({ margin }) => margin || "20px"};
```